app/src/worker.py

```diff
@@ -0,0 +1,106 @@
+import binascii
+import datetime
+import os
+from typing import Optional
+
+from redis import Redis
+
+from common.config import Config
+from common.language import Language
+
+from .app_utils import get_from_and_dkim_domain, scan_and_log
+from .check_results import save_check_results
+from .db import ScanLogEntrySource
+from .logging import build_logger
+from .resolver import setup_resolver
+from .scan import DomainValidationException, ScanningException
+from .translate import translate
+
+LOGGER = build_logger(__name__)
+REDIS = Redis.from_url(Config.Data.REDIS_URL)
+
+setup_resolver()
+
+
+def check_domain_scan_job(
```

Comment:
if it is able to check email messages, it shouldn't be called like that -  why not just "scan_job" calling "scan_message_and_domain" or "scan_domain" ?